src/store/modules/gameList.js

```diff
@@ -30,7 +31,7 @@ export default {
       if (gameIndex < 0 || gameIndex > state.openGames.length) {
         return;
       }
-      const [ startedGame ] = state.openGames.splice(gameIndex, 1);
+      const [startedGame] = state.openGames.splice(gameIndex, 1);
```

Comment:
We should update startedGame's status now that we can track started state
```suggestion
      const [startedGame] = state.openGames.splice(gameIndex, 1);
      startedGame.status = GameStatus.STARTED;
```